docs/units/quantity.rst

```diff
@@ -100,21 +100,40 @@ the quantity:
 .. doctest-requires:: matplotlib
 
     >>> from matplotlib import pyplot as plt
-    >>> plt.figure()
+    >>> plt.figure(figsize=(5,3))
     <...>
     >>> plt.plot([1, 2, 3] * u.m)
     [...]
 
+.. plot::
+    
+    from astropy import units as u
+    from astropy.visualization import quantity_support
+    quantity_support()
+    from matplotlib import pyplot as plt
+    plt.figure(figsize=(5,3))
+    plt.plot([1, 2, 3] * u.m)
+
 Quantities are automatically converted to the first unit set on a
 particular axis, so in the following, the y-axis remains in ``m`` even
-though the second line is given in ``cm``::
```

Comment:
I think the double colon `::` is used to indicate that the next block is code. I think it is not necessary given the `>>>`, but do check that the output documentation looks good.